### ISSUE TITLE
Duration not showing in build summary test grid

### ISSUE THREAD

**Issue by jdshkolnik** [deloitte-solvas/SlackerRunner#15]
SlackerRunner doesn't seem to show duration in the grid for each test even though it has that info.

![image](https://cloud.githubusercontent.com/assets/2047520/24314853/32ca11de-10ba-11e7-9880-00b1aa285954.png)

**Comment by ornatwork** [comment]
It still works fine in VS, did you start seeing this when running on newer VsTfs version ?

![slackertesttime](https://cloud.githubusercontent.com/assets/94155/24467964/9688cc88-1484-11e7-9ef5-6d76dca654f1.jpg)

**Comment by jdshkolnik** [comment]
I believe the runner is populating assembly time but not collection time (see [xUnit schema](https://xunit.github.io/docs/format-xml-v2.html#test)).

**Comment by ornatwork** [comment]
I don't think it has anything to do with xUnit or it's schema.  Each runner collects this information during test run to be able to display it later.  I created a test project using the Microsoft UnitTest framework, it also shows 0 duration for each test when run.  I believe there is something broken in the TFS server version that we are using, this used to work.

![tfs_dev_mstests_2](https://cloud.githubusercontent.com/assets/94155/24632237/a73c80d8-1880-11e7-947f-a6534dedc64d.jpg)

**Comment by ornatwork** [comment]
Here is another sample from a different TFS server, that runs the CI SlackerRunner build, it does show the duration per test.

![slackertesttime_tfs_2](https://cloud.githubusercontent.com/assets/94155/24668298/9ca6b140-1923-11e7-9aea-febf0c6cbb64.jpg)

**Comment by jdshkolnik** [comment]
For the record, that's not a different TFS server showing duration, it's VSTS. Both our DEV and PROD TFS servers (2017 Update 1) manifest the same bug.

**Comment by jdshkolnik** [comment]
I'm reopening so we remember to post this as an issue to the Microsoft VSTS repo.
